spanner/test/opentelemetry/test/ot_traces_test.go

```diff
@@ -71,6 +73,88 @@ func TestSpannerTracesWithOpenTelemetry(t *testing.T) {
 	}
 }
 
+func TestSpanAnnotatedWithXGoogSpannerRequestID_unary(t *testing.T) {
+	testSpanAnnotatedWithXGoogSpannerRequestID(t, "cloud.google.com/go/spanner.BatchCreateSessions")
```

Comment:
The following makes the test slightly more future-proof, as when mux sessions becomes the default, we might not create any regular sessions at all.

```suggestion
	targetSpanName := "cloud.google.com/go/spanner.BatchCreateSessions"
	if isMultiplexEnabled {
		targetSpanName = "cloud.google.com/go/spanner.CreateSession"
	}
	testSpanAnnotatedWithXGoogSpannerRequestID(t, targetSpanName)
```